test/test_images/probe_helper/main.go

```diff
@@ -104,33 +106,43 @@ type envConfig struct {
 	// Environment variable containing an upper bound on the duration between events emitted by the CloudSchedulerSource
 	CloudSchedulerSourcePeriod time.Duration `envconfig:"CLOUDSCHEDULERSOURCE_PERIOD" default:"90s"`
 
-	// Environment variable containing the port which listens to the probe to deliver the event
+	// Environment variable containing the port which listens to the probe to forward events
 	ProbePort int `envconfig:"PROBE_PORT" default:"8070"`
 
-	// Environment variable containing the port to receive the event from the trigger
+	// Environment variable containing the port to receive delivered events
 	ReceiverPort int `envconfig:"RECEIVER_PORT" default:"8080"`
 
-	// Environment variable containing the port to send health checks to
-	HealthCheckerPort int `envconfig:"HEALTH_CHECKER_PORT" default:"8060"`
-
 	// Environment variable containing the maximum tolerated staleness duration
 	MaxStaleDuration time.Duration `envconfig:"MAX_STALE_DURATION" default:"5m"`
 
-	// Environment variable containing the timeout duration to wait for an event to be delivered back
-	TimeoutDuration time.Duration `envconfig:"TIMEOUT_DURATION" default:"30m"`
+	// Environment variable containing the maximum timeout duration to wait for an event to be delivered
+	MaxTimeoutDuration time.Duration `envconfig:"MAX_TIMEOUT_DURATION" default:"2m"`
```

Comment:
I think we should have a defaultTimeout if the request doesn't specify the timeout. We can use the maxTimeout for validation but I think the maxTimeout should probably be much bigger.

Reply:
Good point. I added a default timeout environment variable (2m) and increased the max timeout to 30m.